tools/topology/sof-tgl-sdw-max98373-rt5682.m4

```diff
@@ -39,8 +39,10 @@ DEBUG_START
 # PCM7 ----> volume -----> iDisp3
 # PCM9 ----> volume -----> iDisp4
 # PCM10 <---- volume <---- DMIC01  (dmic 48k capture)
-# PCM12 <---- kpb <---- DMIC16k  (dmic 16k capture)
-
+# PCM12 <---- volume <---- DMIC16k  (dmic 16k capture)
+# PCM13 <---- KPB <---- BufferedMic
+# PCM14 ----> passthrough ----> playback Bluetooth
+# PCM15 <---- passthrough <---- capture Bluetooth
```

Comment:
I don't get what you are trying to do here. The list for SoundWire devices is:

````
PCM10  capture	DMIC
PCM11  capture	DMIC16kHz
PCM12  capture 	BufferedMic (aka KPB)
PCM13  playback Bluetooth
PCM14  capture 	Bluetooth
````

Reply:
I thought the list follows pipeline sequence, since DMIC_PIPELINE_KWD_ID is defined as `13', thus my changes.

```
#
# Define the pipelines
#
# PCM0 <---> volume <----> playback (Headset - ALC5682)
# PCM1 <---> volume <----> capture  (Headset - ALC5682)
# PCM2 ----> smart_amp ----> ALH0x102 (Speaker -max98373)
```

Reply:
So the number here refers to PCM ID (FOO_PCM_BAR_ID macros). I propose we use a single PCM for Bluetooth and pick PCM14 for all Soundwire topologies. So then it would be:
```
# PCM14 ----> passthrough ----> playback Bluetooth
# PCM14 <---- passthrough <---- capture Bluetooth
```

Reply:
Ack, in the current description, PCM9 should be PCM8 instead, PCM 9 is free so I take it, will change to 14 instead, thanks @plbossart and @kv2019i  
```
# PCM5 ----> volume -----> iDisp1
# PCM6 ----> volume -----> iDisp2
# PCM7 ----> volume -----> iDisp3
# PCM9 ----> volume -----> iDisp4
```